tests/Functional/ContentConsultationRenderTest.php

```diff
@@ -29,8 +31,10 @@ class ContentConsultationRenderTest extends ContentRenderTestBase {
     'system',
     'path',
     'field_group',
+    'oe_content_sub_entity_document_reference',
```

Comment:
Since oe_theme_content_document_reference is in the dependency of oe_theme_content_consultation, oe_content_sub_entity_document_reference will be enabled as a dependency of  oe_theme_content_document_reference. Looks like line can be removed.